### ISSUE TITLE
Issue with k8s.io/docs/getting-started-guides/fedora/fedora_manual_config/

### ISSUE THREAD

**Issue by adelton** [kubernetes/website#7521]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**

The page https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/ says:

```

    Edit /etc/kubernetes/kubelet to appear as such:

###
# Kubernetes kubelet (node) config

# The address for the info server to serve on (set to 0.0.0.0 or "" for all interfaces)
KUBELET_ADDRESS="--address=0.0.0.0"

# You may leave this blank to use the actual hostname
KUBELET_HOSTNAME="--hostname-override=fed-node"

# location of the api-server
KUBELET_API_SERVER="--api-servers=http://fed-master:8080"

# Add your own!
#KUBELET_ARGS=""
```

When I do that, attempt to start the kubelet service seemingly passes but `systemctl status kubelet` reports

```
Failed to start Kubernetes Kubelet Server.
```

and `journalctl _COMM=kubelet -l` says

```-- Logs begin at Mon 2018-02-26 12:13:57 UTC, end at Mon 2018-02-26 13:37:59 UTC. --
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]: Error: unknown flag: --api-servers
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]: The kubelet binary is responsible for maintaining a set of containers on a
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]:   particular node. It syncs data from a variety of sources including a
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]:   Kubernetes API server, an etcd cluster, HTTP endpoint or local file. It then
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]:   queries Docker to see what is currently running.  It synchronizes the
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]:   configuration data, with the running set of containers by starting or stopping
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]:   Docker containers.
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]: Usage:
Feb 26 12:34:33 kubernetes-cri-o.example.test kubelet[3707]:   kubelet [flags]
[...]
```

**Proposed Solution:**

Update the documentation to match the current software.

**Page to Update:**
https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/

Kubernetes Version:

```
# rpm -qa 'kubernetes*'
kubernetes-client-1.9.1-5.fc27.x86_64
kubernetes-node-1.9.1-5.fc27.x86_64
kubernetes-master-1.9.1-5.fc27.x86_64
kubernetes-1.9.1-5.fc27.x86_64
```

**Comment by adelton** [comment]
I was able to make things work to the point where the node shows up in the `kubectl get nodes` by creating `/etc/kubernetes/kubelet.kubeconfig` with content

```
apiVersion: v1
kind: Config
clusters:
  - cluster:
      server: http://kubernetes-cri-o.example.test:8080/
    name: local
contexts:
  - context:
      cluster: local
    name: local
current-context: local
```

and adding `--kubeconfig=/etc/kubernetes/kubelet.kubeconfig` to `KUBELET_ARGS` in `/etc/kubernetes/kubelet`.

I took inspiration from https://github.com/kubernetes/kubernetes/pull/40050/files. Maybe the Fedora packaging ought to be changed to ship an example `kubelet.kubeconfig` file and use the `KUBELET_KUBECONFIG` environment variable?
